pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/StartServiceManagerCommand.java

```diff
@@ -69,6 +69,8 @@
   private String _clusterName = DEFAULT_CLUSTER_NAME;
   @Option(name = "-port", required = true, metaVar = "<int>", usage = "Pinot service manager admin port, -1 means disable, 0 means a random available port.")
   private int _port;
+  @Option(name = "-healthCheckAllComponents", metaVar = "<boolean>", usage = "Pinot service manager health check returns the all components health check. The health check returns OK when all the components are returning OK.")
```

Comment:
Perhaps we need an enum instead of boolean. For example: healthCheckComponents=CONTROLLER|BROKER? Don't have a concrete use case in mind, so just a soft suggestion at the moment.

Another thought is `stateCheck` instead of `healthCheck`. That we, it can also be used to wait for `bootstrap` of a table.

Reply:
make sense, adding an option of healthcheck services to be included in the `/health` call.